packages/plugin-typescript/eslint.config.js

```diff
@@ -0,0 +1,21 @@
+import tseslint from 'typescript-eslint';
+import baseConfig from '../../eslint.config.js';
+
+export default tseslint.config(
+  ...baseConfig,
+  {
+    files: ['**/*.ts', '!**/mocks'],
```

Comment:
Other projects don't have `mocks` directory excluded this way. Is it needed? The `**/mocks/fixtures` pattern is globally ignored, BTW.